Makefile

```
@@ -119,9 +120,10 @@ define ENVIRONMENT_EXEC
 			--env INSIDE_ENVIRONMENT=true \
 			--network host \
 			--mount type=bind,source=${CURRENT_DIR},target=/git/vectordotdev/vector \
-			--mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
+			$(if $(findstring docker,$(CONTAINER_TOOL)),--mount type=bind$(COMMA)source=/var/run/docker.sock$(COMMA)target=/var/run/docker.sock,) \
```

Could this be done by escaping the comma with a backslash instead?
```suggestion
			$(if $(findstring docker,$(CONTAINER_TOOL)),--mount type=bind\,source=/var/run/docker.sock\,target=/var/run/docker.sock,) \
```

---

i started out that way, and also tried more `\`, but the docs above `EMPTY` https://www.gnu.org/software/make/manual/make.html#Syntax-of-Functions calls out comma as an example substitution

---

Gotcha, fair enough.